spec/actions_specs/crashlytics_spec.rb

```diff
@@ -98,13 +98,13 @@
             end").runner.execute(:test)
 
             expect(command).to eq([
-              @crashlytics_bundle,
-              "wadus",
-              "secret",
-              "-ipaPath './fastlane/spec/fixtures/fastfiles/Fastfile1'",
-              "-notifications YES",
-              "-debug NO"
-            ])
+                                    @crashlytics_bundle,
+                                    "wadus",
+                                    "secret",
+                                    "-ipaPath './fastlane/spec/fixtures/fastfiles/Fastfile1'",
+                                    "-notifications YES",
+                                    "-debug NO"
+                                  ])
```

Comment:
Indent the right bracket the same as the start of the line where the left bracket is.